lib/docs/token-response-types.md

```diff
@@ -0,0 +1,74 @@
+# Token Response Types
+
+When acquiring tokens, `msal@1.x.x` considers three different response types:
+
+| Response Type | Definition | Reference | Request Example |
+| ------- | -------- | -------- | -------- |
+| **token** | Indicates a request will be made for an Access Token | [OAuth 2.0 Implicit Flow Specification](https://tools.ietf.org/html/rfc6749#section-4.2.1) | [Request example](https://tools.ietf.org/html/rfc6749#section-4.2.1)
+| **id_token** | Indicates a request will be made for an ID Token | [OpenID Connect Specification](https://openid.net/specs/openid-connect-core-1_0.html#Authentication) | [Request Example](https://openid.net/specs/openid-connect-core-1_0.html#id_tokenExample)
+| **id_token token** | Indicates that both an ID Token and Access Token will be requested | [OpenID Connect Specification](https://openid.net/specs/openid-connect-core-1_0.h:tml#Authentication) | [Request Example](https://openid.net/specs/openid-connect-core-1_0.html#id_token-tokenExample)
+
+The table below describes the mapping of `msal@1.x.x` APIs in combination with the user-provided `scopes` to the resulting `response_type` set in the server request.
+
+| Scenarios | Client ID as Only Scope | No Scopes | Access Token Scopes with No client ID | Access Token Scopes with client ID
+| ------- | ------- | -------- | --------- | -------- |
+| **loginRedirect/Popup** | id_token | id_token |id_token token | id_token token
+| **acquireTokenInteractive** | id_token | Error | token | id_token token 
+| **acquireTokenSilent** | id_token | Error | token | id_token token
+| **loginRedirect/Popup with accounts not matching** | id_token | id_token | id_token token | id_token token
+| **acquireTokenInteractive with accounts not matching** | id_token | Error | id_token token | id_token token
```

Comment:
Can we call this `acquireTokenRedirect/Popup` for more clarity

Reply:
Sure

Reply:
@tnorling I removed the acquireTokenSilent rows because they matched popup and redirect in every case, so I went for 'acquireTokenX' in the new version. What do you think?